README.md

```diff
@@ -2,10 +2,10 @@
 Python Command Line Tools for interacting with Multivers<sup>X</sup>.
 
 ## Documentation
-[docs.multiversx.com](https://docs.multiversx.com/sdk-and-tools/erdpy/)
+[docs.multiversx.com](https://docs.multiversx.com/sdk-and-tools/mypy/)
 
 ## CLI
-[CLI](erdpy/CLI.md)
+[CLI](multiversx_sdk_cli/CLI.md)
 
 ## Distribution
 [erdpy-up](https://docs.multiversx.com/sdk-and-tools/erdpy/installing-erdpy/) and [PyPi](https://pypi.org/project/erdpy/#history)
```

Comment:
I think we have a chicken and the egg issue here. Should we go also with the new path?
```suggestion
[erdpy-up](https://docs.multiversx.com/sdk-and-tools/mxpy/installing-mxpy/) and [PyPi](https://pypi.org/project/erdpy/#history)
```
If you will change the previous path, probably this one must be changed as well.
Will come with a PR on docs into this direction.